### DISCUSSION TITLE
GitHub Sponsors and what happens to it when I die (GitHub Successor)

### DISCUSSION THREAD

**Discussion by tomkerkhove** [community/community#4641] [Other Feature Feedback, Questions, & Ideas]
I was thinking about GitHub Sponsors and what happens to it when I die. I've configured a successor but what about my sponsors? Is there anything planned on this front?

What I'd love to have is that when my successor gets access, GitHub will automatically:

1. Allow my successor to change the account that the money receives to either a charity or sponsoring somebody else which I have configured before I passed away
2. Send a notification to all my sponsors that I've passed away and that all money will now transfer to <charity name> and they can decide to support that instead or stop sponsoring

Why is this important? Well, if I pass away my bank account can be closed and my wife will have no clue how to change this where, for example

Context: https://blog.tomkerkhove.be/2021/08/06/thinking-about-your-open-source-legacy/

**Comment by bdresser** [comment]
Thank you for starting this discussion, @tomkerkhove! We don't have immediate plans to introduce a successor feature for Sponsors, but I agree this is important to consider in the long-term as the program grows. We'll keep you posted here with any updates.

**Reply by tomkerkhove** [reply]
Sounds good to me, happy to help where I can.

**Reply by tomkerkhove** [reply]
Just so you know - The successor feature is already there but I agree that in some cases it might be another person since it's related to money.
